container/build_syne_tune_container.sh

```diff
@@ -55,8 +55,10 @@ fi
 docker build -t ${image} . \
              --build-arg REGION=${region} \
              --build-arg DLAMI_REGISTRY_ID=${DLAMI_REGISTRY_ID} \
-             --build-arg CONTEXT=${CONTEXT}
+             --build-arg CONTEXT=${CONTEXT} \
+             --pull \
+             --no-cache
```

Comment:
If I am not mistaken, this would build the image every time we call the remote launcher right?

Reply:
It seems to me the `build_syne_tune_container.sh` is only triggered when the image is not available in the ECR:
https://github.com/awslabs/syne-tune/blob/cc8b7b39192e15d95f2252c77b6d2294775b31ea/syne_tune/remote/remote_launcher.py#L243-L256

Reply:
Indeed you are right. 
So, it wont matter for users as the docker image wont be built/uploaded at each call and if they run the command themselves, then the docker image is updated. 
Thanks for proposing this improvement.